src/sentry/relay/config/__init__.py

```diff
@@ -146,6 +146,11 @@ def get_filter_settings(project: Project) -> Mapping[str, Any]:
             "*https://reactjs.org/docs/error-decoder.html?invariant={418,419,422,423,425}*"
         ]
 
+    enable_chunk_upload_error_filter = project.get_option("filters:chunk-load-error")
+    if enable_chunk_upload_error_filter:
+        # ChunkLoadError: Loading chunk 3662 failed. (error: https://xxx.com/_next/static/chunks/29107295-0151559bd23117ba.js)
+        error_messages += ["ChunkLoadError: Loading chunk * failed. (error: *)"]
```

Comment:
This glob pattern assumes that `type` and `value` for an error always exist and the type is always `ChunkLoadError` for all the errors of this type (e.g., there is no other type with the same error message).